app/assets/javascripts/discourse/app/mixins/uppy-s3-multipart.js

```diff
@@ -18,142 +17,147 @@ export default Mixin.create({
       // chunk size via getChunkSize(file), so we may want to increase
       // the chunk size for larger files
       limit: 10,
-      retryDelays,
+      retryDelays: this.retryDelays,
 
-      createMultipartUpload(file) {
-        self._uppyInstance.emit("create-multipart", file.id);
+      createMultipartUpload: this._createMultipartUpload.bind(this),
+      prepareUploadParts: this._prepareUploadParts.bind(this),
+      completeMultipartUpload: this._completeMultipartUpload.bind(this),
+      abortMultipartUpload: this._abortMultipartUpload.bind(this),
```

Comment:
Using `@bind` decorator on methods is preferred instead

Reply:
Huh TIL, nice!